### PR TITLE
Addition of Skill Passport

### PR DESCRIPTION
**Pull request by rohitsamagra**

### CODE REVIEW

File: reference-solution-for-digital-credentials/Skill Passport/Architecture of Skill Passport.md [hunk 1]
```diff
@@ -0,0 +1,261 @@
+# Architecture Diagram
+
+<div data-full-width="true">
+
+![image](https://github.com/rohitsamagra/community/assets/145356240/4a256b10-b25b-4ec9-af2b-51b7f53868d8)
```
Comment (srprasanna) [inline reference-solution-for-digital-credentials/Skill Passport/Architecture of Skill Passport.md]: Can you move this to the repo.

**Review by srprasanna — COMMENTED** [review]
@rohitsamagra can you link Architecture of Skill passport as a sub-page of Value of Skill passport. Having them as two different pages might not work. Is the solution based on RC 1.0? Can you point to Skill Passport documentation for further information on this reference solution.

File: reference-solution-for-digital-credentials/Skill Passport/Architecture of Skill Passport.md (hunk 1 above)
Comment (rohitsamagra) [reply reference-solution-for-digital-credentials/Skill Passport/Architecture of Skill Passport.md]: Done

**Comment by rohitsamagra** [comment]
> @rohitsamagra can you link Architecture of Skill passport as a sub-page of Value of Skill passport. Having them as two different pages might not work. Is the solution based on RC 1.0? Can you point to Skill Passport documentation for further information on this reference solution.

- Moved Architecture of Skill passport within Value of skill passport
- Yes it is based on RC 1.0
- This is the Github repo at our end of Skill passport: https://github.com/COMPASS-DPG/passbook

Please let me know if I can provide any more information. Thanks

**Pull request merged**